lib/scala/runtime-version-manager/src/main/scala/org/enso/runtimeversionmanager/runner/Runner.scala

```diff
@@ -232,13 +232,19 @@ class Runner(
       case None =>
         runtimeVersionManager.withEngineAndRuntime(engineVersion) {
           (engine, runtime) =>
-            val ensoLauncher = Option(System.getenv(Runner.LAUNCHER_ENV_NAME))
-            val requiresJVMRunner =
-              ensoLauncher.exists(_.equals("shell")) || runSettings.jvmMode
-            if (requiresJVMRunner) {
+            if (runSettings.jvm.isDefined) {
```

Comment:
Is there a recommended way to (unit) test this logic?

Reply:
I don't think we are testing this at all. Make sure you try manually launching `project-manager`.